### PR TITLE
feat(NDIV-250): Added how courts will contact you page

### PR DESCRIPTION
**Pull request by SebFlippence**
### JIRA link ###

[NFDIV-250](https://tools.hmcts.net/jira/browse/NFDIV-250) English version - How the court will contact you

### Change description ###

Added "How the court will contact you" page and logic for email, required checkboxes and phone number validation.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/app/case/answers/getAnswerRows.ts [hunk 1]
```diff
@@ -12,10 +12,12 @@ export const getAnswerRows = function (section: Sections): GovUkNunjucksSummary[
     language,
     isDivorce,
     formState,
+    userEmail,
   }: {
     language: 'en' | 'cy';
     isDivorce: boolean;
     partner: string;
+    userEmail?: string;
```
Comment (linusnorton) [inline src/main/app/case/answers/getAnswerRows.ts:20]: does it have to be optional?

File: src/main/app/form/validation.ts [hunk 2]
```diff
@@ -88,3 +88,13 @@ export const isInvalidHelpWithFeesRef: Validator = value => {
     }
   }
 };
+
+export const isPhoneNoValid: Validator = value => {
+  if (!value) {
```
Comment (linusnorton) [inline src/main/app/form/validation.ts:93]: is this necessary? what case does it catch?

File: src/main/steps/common/common.content.ts [hunk 3]
```diff
@@ -158,7 +158,8 @@ export const generatePageContent = (
   language: Language,
   pageContent?: TranslationFn,
   isDivorce = true,
-  formState?: Partial<Case>
+  formState?: Partial<Case>,
+  userEmail?: string
```
Comment (linusnorton) [inline src/main/steps/common/common.content.ts:162]: as above, I'm wondering when it's not set?

**Review by velimd — APPROVED** [review]

File: src/main/app/case/answers/getAnswerRows.ts (hunk 1 above)
Comment (SebFlippence) [reply src/main/app/case/answers/getAnswerRows.ts]: 👍 Good spot, I've made this required in e21275c.

File: src/main/app/form/validation.ts (hunk 2 above)
Comment (SebFlippence) [reply src/main/app/form/validation.ts]: 👍 I've moved this falsy check into the RegEx in 6b31e94.

File: src/main/steps/common/common.content.ts (hunk 3 above)
Comment (SebFlippence) [reply src/main/steps/common/common.content.ts]: 👍 Good spot, I've made this required in e21275c.

**Review by linusnorton — APPROVED** [review]

**Pull request merged**